documentation/docs/10-adapters.md

```diff
@@ -93,6 +93,7 @@ Within the `adapt` method, there are a number of things that an adapter should d
   - Imports `App` from `${builder.getServerDirectory()}/app.js`
   - Instantiates the app with a manifest generated with `builder.generateManifest({ relativePath })`
   - Listens for requests from the platform, converts them to a standard [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) if necessary, calls the `render` function to generate a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) and responds with it
+  - `render` optionally accepts a `platform` object, which allow passing of environment specific values and functions. The `platform` object will become be accessable from the handle hook, and any API endpoints in the application
```

Comment:
Just a couple minor spelling/grammar changes:

`render` optionally accepts a `platform` object, which allows passing of environment specific values and functions. The `platform` object will be accessible from the handle hook and any API endpoints in the application